config/rbac/role.yaml

```diff
@@ -31,8 +31,8 @@ rules:
   - clusterversions
   verbs:
   - get
-  - watch
   - list
+  - watch
```

Comment:
- Maybe one of the previous PRs was manually edited and so is the change when roles are generated via `make bundle` now.